framework/src/base/MooseVariableScalar.C

```diff
@@ -23,6 +23,9 @@
 #include "libmesh/numeric_vector.h"
 #include "libmesh/dof_map.h"
 
+// C++
```

Comment:
I still disagree, but I'd rather get this merged than argue, so I'll push one more commit stripping those two comments just as soon as these retests finish.

That said,
```
roystgnr@zappa:(scalar_scaling_fix) ~/git/moose/modules
$ git blame ../framework/src/base/MooseVariableScalar.C | tail -n +22 | head -n 5
e4501c55 (David Andrs       2012-02-06 08:48:49 +0000  22) // libMesh
ff8a0bdc (Cody Permann      2013-02-07 12:33:07 +0000  23) #include "libmesh/numeric_vector.h"
02f63040 (John W. Peterson  2015-11-03 10:18:43 -0700  24) #include "libmesh/dof_map.h"
e4501c55 (David Andrs       2012-02-06 08:48:49 +0000  25) 
b99fd8ac (Roy H. Stogner    2017-07-05 17:16:21 -0500  26) // C++
```

https://en.wikipedia.org/wiki/I_learned_it_by_watching_you!   ;-)